### ISSUE TITLE
How to load a .vue file?

### ISSUE THREAD

**Issue by AbrahamBrookes** [GeekyAnts/vue-native-website#68]
I have figured this out myself, but the documentation needs to be updated.

The [documentation](https://vue-native.io/docs/installation.html#modifying-app-json) states:

> At this point you will need to manually change the app.json that has been generated inside the new project directory. To make the packager recognize .vue files, a sourceExts field under packagerOpts.

    {
      "expo": {
        "name": "Your app's name",
        "platforms": [
          "ios",
          "android",
          "web"
        ],
        "version": "1.0.0",
        ...
        "packagerOpts": {
         "sourceExts": ["js", "json", "ts", "tsx", "vue"],
          "config": "metro.config.js"
        }
      }
    }

It then goes on to say:
>Remove the Content of the `App.js` file and rename `App.js` file with `App.vue`.

This is misleading and backwards. Let's describe how the system actually works.

The `sourceExts` property in `App.json` is **order-sensitive**. You need to put the `"vue"` entry **at the start of the list** like this:

    "sourceExts": ["vue", "js", "json", "ts", "tsx"],

Then, don't do some crazy delete content rename file bs. Just **delete App.js**. `App.vue` already exists by default in the initial installation - create it if necessary.

You may need to restart the Metro server process (in your shell, ctrl+c to exit the program, then `npm start` again). And bam, up comes the vue app.

Devs please update your docs - there appears to be a lot of out of date information. Also please run through your `vue-native init` function - right now it installs a **react**-native app by default and doesn't consider the vue file that is included in the install.

**Comment by RishabhKarnad** [comment]
@AbrahamBrookes @wyxos please provide information about the platform (OS, mobile platform). @wyxos you seem to be using Vue Native on web, which doesn't work yet

https://github.com/GeekyAnts/vue-native-cli/issues/37

**Comment by AbrahamBrookes** [comment]
I'm not using web, this is using the mobile qr-scan thing on an Android phone. And I may have done some other stuff, but this was from a largely fresh install directly from the cli tool, as outlined in the docs. 

The way I got to this solution was by looking at the demo code for the kitchen sink app on github and trying to replicate the structure ther, which is where I saw that `vue` is at the top of their `sourceExts` list.
